index.html

```diff
@@ -0,0 +1,196 @@
+                    <h5>Kina Mountain</h5>
+                    <span>
+                        <img src="images/logos/location.png" alt="Location"><a href="#">Cambodia</a>
+                    </span>
+                </div>
+                <div class="destination_block">
+                    <img src="images/image/kina-mountain2.png" alt="picture">
+                    <span class="rating"><img src="images/logos/star.png" alt="star">4.2</span>
+                    <h5>Kina Mountain</h5>
+                    <span>
+                        <img src="images/logos/location.png" alt="Location"><a href="#">Cambodia</a>
+                    </span>
+                </div>
+                <div class="destination_block">
+                    <img src="images/image/kina-mountain3.png" alt="picture">
+                    <span class="rating"><img src="images/logos/star.png" alt="star">3.5</span>
+                    <h5>Kina Mountain</h5>
+                    <span>
+                        <img src="images/logos/location.png" alt="Location"><a href="#">Cambodia</a>
+                    </span>
+                </div>
+            </div>
+        </div>
+        <div class="reviews">
+            <div class="review_block">
+                <img class="background_img" src="images/logos/cquote.png" alt="Cquote">
+                <p class="review">Our trip to Morocco was truly a onece in a lifetime experience and we are so grateful to everyone that made it happen seamlessly. Our travel planner, Jaouad, was increadible.</p>
+                <p>-Vand D</p>
+                <p>Happy Treloo</p>
+                <img class="review_img" src="images/image/girl.png" alt="A photo">
+                <span class="review_rating"><img src="images/logos/star.png" alt="star">4.5</span>
+            </div>
+            <div class="review_block">
+                <img class="background_img" src="images/logos/cquote.png" alt="Cquote">
+                <p class="review">Our trip to Morocco was truly a onece in a lifetime experience and we are so grateful to everyone that made it happen seamlessly. Our travel planner, Jaouad, was increadible.</p>
+                <p>-Tru Vio</p>
+                <p>Happy Treloo</p>
+                <img class="review_img" src="images/image/boy.png" alt="A photo">
+                <span class="review_rating"><img src="images/logos/star.png" alt="star">4.9</span>
+            </div>
+            <span>
+                <input type="button" value="<">
+                <input type="button" value=">">
+            </span>
+        </div>
+        <div class="promotion">
+            <p>Don’t Miss The 50% Discount if You register Today</p>
+            <button>Register Today</button>
+        </div>
+    </main>
+    <footer>
+        <p class="annotation">© 2022 Treloo LLC - All right reserved.</p>
+        <nav class="navbar_bottom">
+            <ul>
+                <li><a href="">Review</a></li>
```

Comment:
Check code one more time, if there are similar things that we discussed. Like # here

Reply:
done